### PR TITLE
Password verification

### LINKED ISSUE #985: Remove password verification
### Description of problem
A [test in cockpit-composer](https://github.com/osbuild/cockpit-composer/blob/6e735255bb114be4ee5687d354aaec6fe3d67bd1/test/verify/check-custom#L173) regarding password verification is currently failing in Fedora but passing in RHEL. But considering the use cases that cockpit-composer supports, requiring an acceptable password based on the host machine isn't a valid use case. The proposed solution is to drop the password verification, to avoid blocking the user from defining the password that they want to use.

### Steps to Reproduce

1. Navigate to the Blueprint page.
2. Click the "Create User Account" button on the Customizations tab.
3. Specify a weak password, like "foobar", in both password fields.

### Actual results
A message displays, saying "Password quality check failed: The password is shorter than 8 characters" and you are prevented from creating the user account.

### Expected results
The user should not be blocked when specifying a weak password. Additionally, the password verification is checking the password policy on the host machine, which may or may not be the password policy of the target machine for the image you're building. Therefore, I'd propose we remove the password verification altogether, and just check that the passwords match.

cc @jkozol @henrywang

### PR DESCRIPTION
**Pull request by jgiardino**
This removes the password quality check from the Create User Account modal.

_I forgot to include in the original commit:_
fixes #985
🤦

### CODE REVIEW

**Comment by jgiardino** [comment]
@henrywang @jkozol Do we also want to handle the password verification test as part of this PR? Please let me know if you have any suggestions on updates for tests that are needed.

File: components/Form/Password.js [hunk 1]
```diff
@@ -1,56 +1,28 @@
...
-      // if both values match, and will pass the password to the parent component
-    } else {
-      this.setState({
-        passwordQuality: "",
-        warningQuality: ""
-      });
-    }
     if (event.target.value === this.state.passwordTwo) {
       this.setState({ displayWarningMatch: false });
       if (event.target.value === "") {
```
Comment (jkozol) [inline components/Form/Password.js:28]: The Create button is not disabled when only the first password is entered. Adding the else statement will invalidate the password unless both passwords are empty.

createButtonEnabled

Comment (jkozol) [reply components/Form/Password.js:28]:
```suggestion
      if (event.target.value === "") {
        this.props.setValidPassword(event.target.value);
      }
    } else {
      this.props.setValidPassword();
    }
```

File: components/Form/Password.js [hunk 2]
```diff
@@ -64,21 +36,15 @@ class Password extends React.Component {
     if (this.state.passwordOne === event.target.value) {
       this.setState({ displayWarningMatch: false });
     }
-    // if the passwords match, and there's no quality warning message defined
-    // then pass the password to the parent component, otherwise, the password values
-    // are invalid and the submit button will be disabled
-    if (this.state.passwordOne === event.target.value && this.state.warningQuality === "") {
+    if (this.state.passwordOne === event.target.value) {
       this.props.setValidPassword(event.target.value);
```
Comment (jkozol) [inline components/Form/Password.js:40]: These if statements can be combined.

**Comment by jkozol** [comment]
@jgiardino Can you review the changes I added per my comments?

**Pull request merged**